<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyGoodsNumDlg</class>
 <widget class="QDialog" name="ModifyGoodsNumDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>646</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModifyGoodsNumDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ModifyGoodsNumDlg {
	border-image: url(:/shareImages/dlg_background.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="TitleBarWidget" name="titleBarWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>315</width>
       <height>340</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>14</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>13</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_4" native="true">
        <property name="minimumSize">
         <size>
          <width>284</width>
          <height>305</height>
         </size>
        </property>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>291</width>
           <height>91</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>284</width>
           <height>87</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(250, 250, 250);</string>
         </property>
         <widget class="QLabel" name="goodsNameLabel">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>23</y>
            <width>70</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#goodsNameLabel {
	font-size: 14px;
	color: rgb(51, 51, 51);
}</string>
          </property>
          <property name="text">
           <string>商品名称：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="goodsNameValueLabel">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>20</y>
            <width>150</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#goodsNameValueLabel {
	font-size:14px;
	color: rgb(51, 51, 51);
}
</string>
          </property>
          <property name="text">
           <string>无籽大西瓜</string>
          </property>
         </widget>
         <widget class="QLabel" name="oldNumLabel">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>52</y>
            <width>70</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#oldNumLabel {
	font-size:14px;
	color: rgb(51, 51, 51);
}
</string>
          </property>
          <property name="text">
           <string>原数量：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="oldNumValueLabel">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>50</y>
            <width>150</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#oldNumValueLabel {
	font-size:16px;
	color: rgb(51, 51, 51);
}
</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>90</y>
           <width>291</width>
           <height>218</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>284</width>
           <height>218</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <widget class="QLabel" name="newNumLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>21</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#newNumLabel {
	font-size:14px;
	color: rgb(51, 51, 51);
}
</string>
          </property>
          <property name="text">
           <string>现数量：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="newNumEdit">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>21</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#newNumEdit {
	font-size:14px;
	color: rgb(51, 51, 51);
	padding-left:10px;
}
</string>
          </property>
         </widget>
         <widget class="QPushButton" name="commitButton">
          <property name="geometry">
           <rect>
            <x>19</x>
            <y>156</y>
            <width>245</width>
            <height>43</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>245</width>
            <height>43</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#commitButton {
	color: rgb(255, 255, 255);
	font-size:16px;
	border-image: url(:/shareImages/sure_normal.png);
}

QPushButton#commitButton:pressed {
	border-image:url(:/shareImages/sure_press.png)
}
</string>
          </property>
          <property name="text">
           <string>提交</string>
          </property>
         </widget>
         <widget class="QLabel" name="newAmountLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#newAmount {
	font-size:14px;
	color: rgb(51, 51, 51);
}
</string>
          </property>
          <property name="text">
           <string>金额：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="newAmountEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>80</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#newAmountEdit {
	font-size:14px;
	color: rgb(51, 51, 51);
	padding-left:10px;
}
</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>340</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>325</width>
       <height>340</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>14</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="WndDigitalKeyboard" name="DigitalKeyWnd" native="true">
        <property name="minimumSize">
         <size>
          <width>303</width>
          <height>303</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WndDigitalKeyboard</class>
   <extends>QWidget</extends>
   <header location="global">WndDigitalKeyboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
